backend/src/util/VerifyFIN.test.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
+import validateFIN from "./VerifyFIN"
+
+test('Valid FIN', () => {
+    expect(validateFIN("F1234567N")).toBeTruthy
+})
+
+test('Lowercase first alphabet', () => {
+    expect(validateFIN("f3725759K")).toBeTruthy
```

Comment:
The assertions need to be called, e.g. `toBeTruthy()`.

Reply:
Got it!